things/database.py

```diff
@@ -754,12 +754,8 @@ def make_search_filter(query: str) -> str:
 
     Example:
     --------
-    >>> make_search_filter('dinner')
-    'AND (
-        TASK.title LIKE "%dinner%"
-        OR TASK.notes LIKE "%dinner%"
-        OR AREA.title LIKE "%dinner%"
-    )'
+    >>> make_search_filter('dinner') #doctest: +REPORT_NDIFF
```

Comment:
The docstring are for humans to read, not machines. The "#doctest" makes it harder to read.

Reply:
Wasn't supposed to be committed. Will remove it in a new pull request.